backend/capellacollab/sessions/hooks/jupyter.py

```diff
@@ -74,14 +74,20 @@ def post_session_creation_hook(
         user: users_models.DatabaseUser,
         **kwargs,
     ):
+        host = (
+            self._jupyter_public_uri.hostname
+            if self._jupyter_public_uri.hostname
+            else ""
+        )
+
         operator.create_public_route(
             session_id=session_id,
-            host=self._jupyter_public_uri.hostname,
+            host=host,
```

Comment:
```suggestion
            host=self._jupyter_public_uri.hostname or "",
```

And remove the `host` variable assignment above,

Reply:
Thats way nice!